src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -108,6 +109,37 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
     }
   }
 
+  @Path("/{workspaceNamespace}/{workspaceName}")
+  @ApiOperation(value = "Update attributes of a workspace",
+    nickname = "update workspace",
+    httpMethod = "Patch",
+    produces = "application/json",
+    response = classOf[Workspace])
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
+    new ApiImplicitParam(name = "workspaceName", required = true, dataType = "string", paramType = "path", value = "Workspace Name"),
+    new ApiImplicitParam(name = "entityType", required = true, dataType = "string", paramType = "path", value = "Entity Type"),
+    new ApiImplicitParam(name = "entityName", required = true, dataType = "string", paramType = "path", value = "Entity Name"),
+    new ApiImplicitParam(name = "entityUpdateJson", required = true, dataType = "org.broadinstitute.dsde.rawls.workspace.EntityUpdateOperations$EntityUpdateOperation", paramType = "body", value = "Update operations")
```

Comment:
Missed a rename: should be `org.broadinstitute.dsde.rawls.workspace.AttributeUpdateOperations.AttributeUpdateOperation`

The Swagger for `updateEntityRoute` also needs this fix.